test/lib/next-test-utils.ts

```diff
@@ -890,17 +890,30 @@ export async function hasErrorToast(
   return (
     (await browser.eval(() => {
       return Boolean(
-        [].slice.call(document.querySelectorAll('nextjs-portal')).find((p) =>
-          p.shadowRoot.querySelector(
-            // TODO(jiwon): data-nextjs-toast may not be an error indicator in new UI
-            isNewDevOverlay ? '[data-issues]' : '[data-nextjs-toast]'
```

Comment:
This won't work, which will always throw as `isNewDevOverlay` can't be accessed in `browser.eval`